src/test/java/madalinapopescu/tests/tema5/demoqa/Buttons.java

```diff
@@ -0,0 +1,23 @@
+package madalinapopescu.tests.tema5.demoqa;
+
+import madalinapopescu.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Buttons {
+    public static void main(String[] args) {
+        goToButtons();
+    }
+
+    public static void goToButtons() {
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement buttons = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[5]"));
```

Comment:
When using xpath as a selector try to avoid using absolute xpath(meaning that you start from the html tag) and use relative xpath(meaning that you'll try to minimize the xpath by starting with the closest unique parent and continuing from there) instead. For example "/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[5]" would become "//div[@class='element-list collapse show']/ul/li[5]"

Reply:
class is not with capital letter there. it seems that there is a user with this username

Reply:
I don't understand what I have to do here